etc/helm/pachyderm/templates/pachd/deployment.yaml

```diff
@@ -64,10 +64,6 @@ spec:
           value: {{ required "postgresql username required" .Values.global.postgresql.postgresqlUsername | quote }}
         - name: POSTGRES_DATABASE
           value: {{ required "postgresql database name required" .Values.global.postgresql.postgresqlDatabase | quote }}
-        {{- if .Values.global.postgresql.ssl }}
```

Comment:
Do we use postgres SSL in pachd? I had a look in the code, we read in the env var, but it's not clear we do anything with it.